src/constants/constants.ts

```diff
@@ -187,3 +188,6 @@ export const azureFunctionsExtensionNotFound = localize('azureFunctionsExtension
 export const installAzureFunction = localize('install', 'Install');
 export const learnMore = localize('learnMore', 'Learn more');
 export const doNotInstall = localize('doNotInstall', 'Do not install');
+export const createProject = localize('createProject', 'Create Azure Function Project');
+export const selectFolder = localize('selectFolder', 'Select Folder for Azure Function Project');
```

Comment:
`Select folder for the Azure Function project`

I'd also suggest having the var be `selectAzureFunctionProjFolder` so it's clearer what the string is when reading the code where it's used